### PR TITLE
Transaction page

### PR DESCRIPTION
**Pull request by niteshn1**

### CODE REVIEW

**Comment by niteshn1** [comment]
Screenshot 2024-03-08 113913

File: src/pages/Transactions.jsx
Comment (souravmsi) [inline src/pages/Transactions.jsx:1]: 1. remove nested component (Status component). Define this component outside.
2. take a pull from master and add search box component

**Review by souravmsi — CHANGES_REQUESTED** [review]
check comments!

**Review by souravmsi — APPROVED** [review]
Approved!!

**Pull request merged**